core/app/[locale]/(default)/_components/organization-schema.tsx

```
@@ -0,0 +1,37 @@
+    },
+    brand: {
+      '@type': 'Brand',
+      name: 'Ryddo'
+    },
+    category: 'Electric Vehicles',
+    keywords: 'electric bikes, e-bikes, electric bicycles, sustainable transportation, commuter bikes, mountain bikes'
+  };
+
+  return (
+    <script
+      dangerouslySetInnerHTML={{ __html: JSON.stringify(organizationSchema) }}
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)